compiler/src/dmd/builtin.d

```diff
@@ -167,6 +173,24 @@ Expression eval_unimp(Loc loc, FuncDeclaration fd, Expressions* arguments)
     return null;
 }
 
+Expression eval_ctfeWrite(Loc loc, FuncDeclaration fd, Expressions* arguments)
```

Comment:
This could be `extern(C++)`, then individual compiler vendors can implement their own version.

Reply:
Ignore this, I didn't see who the caller was.